web/annotation_layer_builder.js

```diff
@@ -147,6 +174,29 @@ class AnnotationLayerBuilder {
     }
     this.div.hidden = true;
   }
+
+  #updatePresentationModeState(state) {
+    if (!this.div) {
+      return;
+    }
+    let disableFormElements = false;
+
+    switch (state) {
+      case PresentationModeState.FULLSCREEN:
+        disableFormElements = true;
+        break;
+      case PresentationModeState.NORMAL:
+        break;
+      default:
+        return;
+    }
+    for (const section of this.div.childNodes) {
+      if (section.hasAttribute("data-internal-link")) {
+        continue;
+      }
+      section.inert = disableFormElements;
```

Comment:
I think the attribute must be removed when `false`.

Reply:
The current code seems to work, but I can certainly change this if you prefer that.

Reply:
Yeah, I just tested and it's working like this, so I'm ok with it.